contributing/code_of_conduct_concrete_example_document.rst

```diff
@@ -0,0 +1,16 @@
+Code of Conduct: Concrete Example Document
+==========================================
+This is a living document that serves to give concrete examples to unwanted behaviour. These examples have all taken place somewhere in the PHP community in the past, and are clear code of conduct violations according to the Symfony code of conduct.
```

Comment:
Documentation lines are usually limited to 72 characters or somewhere near 👍 and headers are followed by a single empty line (sphinxdoc can be picky about this 😄 )

http://symfony.com/doc/current/contributing/documentation/standards.html

Reply:
Changed :)